### PR TITLE
Feat/trust account ethereum

### LINKED ISSUE #117: Tutorial with Remix IDE
For Quorum & Remix for reference [jpmorganchase/quorum/issues/549](https://github.com/jpmorganchase/quorum/issues/549).

### PR DESCRIPTION
**Pull request by akegaviar**
Added MetaMask + Remix.
Added a screenshot on Remix injected web3.
Fixed all screenshots that had a typo.
Added crosslinks.
Added Remix to Tools.

Closes #117

### CODE REVIEW

File: docs/operations/ethereum/using-metamask-with-chainstack.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Using MetaMask with Chainstack
```
Comment (easeev) [inline docs/operations/ethereum/using-metamask-with-chainstack.md]: 1. IMO we can get rid of screenshots and put it to ethereum/tools#interaction-tools
2. We can also have some more elaborate article in KB

File: docs/tutorials/food-supply-temperature-control-on-quorum.md [hunk 2]
```diff
@@ -2,8 +2,11 @@
```
Comment (easeev) [inline docs/tutorials/food-supply-temperature-control-on-quorum.md]: Why are these changes in Ethereum PR?

File: docs/tutorials/trust-fund-account-on-ethereum.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    }
+}
+```
+
+This is your Trust Fund account smart contract:
+
+* The contract belongs to the Ethereum account that deploys the contract through:
+
+``` js
+contract TransferableTrustFundAccount {
```
Comment (easeev) [inline docs/tutorials/trust-fund-account-on-ethereum.md]: This code section looks strange as there's no closing bracket

File: docs/tutorials/trust-fund-account-on-ethereum.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+In your MetaMask, fund each account with Ropsten ether by clicking **Deposit** > **Test Faucet** > **Get Ether**.
+
+### Create and compile the Trust Fund smart contract
+
+1. Open [Remix IDE](https://remix.ethereum.org/)
+
+2. On the home page, click **Environments** > **Solidity**:
+
+![Environment Solidity](./assets/trust-fund-account-on-ethereum/remix-home-solidity.png)
```
Comment (easeev) [inline docs/tutorials/trust-fund-account-on-ethereum.md]: This screenshot is too large and potentially redundant

File: docs/tutorials/trust-fund-account-on-ethereum.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+3. On the left pane, click **File explorers** > plus sign:
+
+![Add .sol](./assets/trust-fund-account-on-ethereum/remix-plus-sign.png)
```
Comment (easeev) [inline docs/tutorials/trust-fund-account-on-ethereum.md]: Same as above

File: docs/operations/ethereum/using-metamask-with-chainstack.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/ethereum/using-metamask-with-chainstack.md]: done

File: docs/tutorials/food-supply-temperature-control-on-quorum.md (hunk 2 above)
Comment (akegaviar) [reply docs/tutorials/food-supply-temperature-control-on-quorum.md]: They absolutely shouldn't be here. My bad. Reverted.

File: docs/tutorials/trust-fund-account-on-ethereum.md (hunk 3 above)
Comment (akegaviar) [reply docs/tutorials/trust-fund-account-on-ethereum.md]: Fixed

File: docs/tutorials/trust-fund-account-on-ethereum.md (hunk 4 above)
Comment (akegaviar) [reply docs/tutorials/trust-fund-account-on-ethereum.md]: Done

File: docs/tutorials/trust-fund-account-on-ethereum.md (hunk 5 above)
Comment (akegaviar) [reply docs/tutorials/trust-fund-account-on-ethereum.md]: Done

**Pull request merged**